app/controllers/api/v2/courses_controller.rb

```diff
@@ -178,6 +178,9 @@ def update_subjects
         return if subject_ids.nil?
 
         @course.subjects = Subject.where(id: subject_ids)
+        generate_course_title_service = Courses::GenerateCourseTitleService.new
+        @course.name = generate_course_title_service.execute(course: @course)
+        @course.save
```

Comment:
don't think we should be calling `.save` here, it'll get called elsewhere and this method isn't related to saving :eyes: 
```suggestion
```

Reply:
Done